<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditPath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Range</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxIndex">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameView">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayoutImages"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBrowse"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionScaled"/>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="AnalView.qrc">
     <normaloff>:/Icons/Remove.png</normaloff>:/Icons/Remove.png</iconset>
   </property>
   <property name="text">
    <string>Clear images</string>
   </property>
   <property name="toolTip">
    <string>Clear images</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="AnalView.qrc">
     <normaloff>:/Icons/Folder.png</normaloff>:/Icons/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="toolTip">
    <string>Browse images folder</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="AnalView.qrc">
     <normaloff>:/Icons/Find.png</normaloff>:/Icons/Find.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select images</string>
   </property>
  </action>
  <action name="actionScaled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AnalView.qrc">
     <normaloff>:/Icons/Resize.png</normaloff>:/Icons/Resize.png</iconset>
   </property>
   <property name="text">
    <string>Scaled</string>
   </property>
   <property name="toolTip">
    <string>Scale images to content</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AnalView.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxIndex</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
